rpc/tests/p2p.rs

```diff
@@ -0,0 +1,267 @@
+use crate::utils::client::{new_test_client, AuthLevel};
```

Comment:
```suggestion
#![cfg(feature = "p2p")]
use crate::utils::client::{new_test_client, AuthLevel};
```
to make sure this runs ok:
```
cd rpc && cargo test --no-default-features
```